ui/main/src/app/api/currentusercard.api.ts

```
@@ -0,0 +1,169 @@
+                self.currentUserCard.setSelectedRecipients(recipients);
+            },
+
+            setSelectedRecipientsForInformation: function (recipients) {
+                self.currentUserCard.setSelectedRecipientsForInformation(recipients);
+            }
+        };
+
+        // prevent unwanted modifications from templates code
+        Object.freeze(opfab.currentUserCard);
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)